domain/src/main/java/org/oppia/android/domain/onboarding/testing/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+"""
+Domain definitions for onboarding testing.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
```

Comment:
This & below:
- Please update the names to be typed out
- Please mark them as testonly (i.e. by adding ``testonly = True``)